common/src/transaction/cosmos_sdk/parser/structs/cosmos_raw_msg.rs

```diff
@@ -410,7 +409,7 @@ impl CosmosRawNormalMsg {
     }
 }
 
-#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
```

Comment:
if `Eq` is now derived in common, the clippy ignore can be removed?

Reply:
Deleted `#![allow(clippy::derive_partial_eq_without_eq)]` in `common/src/lib.rs`.